test/Microsoft.AspNetCore.Mvc.FunctionalTests/FiltersTest.cs

```diff
@@ -554,5 +554,53 @@ public FiltersTest(MvcTestFixture<FiltersWebSite.Startup> fixture)
                 $"CurrentCulture:{expected},CurrentUICulture:{expected}",
                 await response.Content.ReadAsStringAsync());
         }
+
+        [Fact]
+        public async Task RequestSizeLimit_ThrowsForInvalidSize()
```

Comment:
Yeah I tried and was unable to figure out how to initialize the feature for the functional tests. Updating the PR with just unit tests.